.github/workflows/tag-and-release.yaml

```
@@ -26,11 +26,16 @@ jobs:
   publish-package:
     needs: tag-new-version
     if: ${{ needs.tag-new-version.outputs.release_created == 'true' }}
-    runs-on: ubuntu-latest
-    name: Publish ${{ matrix.flavor }}
+    # TODO: (@WSTARR) since we don't run the tests on arm currently we don't need to use the 8 core
+    runs-on: ${{ matrix.architecture == 'arm64' && 'uds-ubuntu-arm64-4-core' || 'uds-ubuntu-big-boy-8-core' }}
+    name: Publish ${{ matrix.flavor }} ${{ matrix.architecture }}
```

```suggestion
    name: Publish ${{ matrix.flavor }} ${{ matrix.architecture }}
    timeout-minutes: 40
```
One more because these are bigger paid runners